src/Serialize.cpp

```diff
@@ -86,8 +86,8 @@ static bool is_syntactic_name(const char *name)
 	try 
 	{
 		auto keyword_regex = regex(keywords);
-		if (!regex_match(name, keyword_regex))
-			return false;
+		if (regex_match(name, keyword_regex))
+			return false; // is keyword
```

Comment:
It was there before I did the refactoring to <regex>. Means, that the passed name is R keyword and thats why it's not a syntactic name (and has to be enclosed in backticks).